src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -174,6 +174,9 @@ export class CalciteAccordionItem {
   updateActiveItemOnChange(event: CustomEvent): void {
     this.requestedAccordionItem = event.detail
       .requestedAccordionItem as HTMLCalciteAccordionItemElement;
+    if (this.el.parentNode !== this.requestedAccordionItem?.parentNode) {
```

Comment:
@Elijbet Can you double check if there's a scenario where `requestedAccordionItem` could be undefined? I'd like to avoid unnecessary checks if possible.

Reply:
Sure.